code/graph-algorithms/eulerian_path/eulerian.py

```diff
@@ -1,3 +1,5 @@
+# Part of Cosmos by OpenGenus Foundation
```

Comment:
yes this is fine ! 👍

Reply:
I have added the comments.

Reply:
@AdiChat  please get this PR reviewed #1682